indigo/distribution.yaml

```diff
@@ -12053,6 +12053,19 @@ repositories:
       url: https://github.com/mikeferguson/simple_grasping.git
       version: master
     status: developed
+  simple_voice-release:
```

Comment:
The git repo names are good as the are but this line should not have the `-release` suffix. With that suffix being removed the `packages` entry under `release` is obsolete and should be removed too.